server/sonar-process/src/main/java/org/sonar/process/systeminfo/JvmStateSection.java

```diff
@@ -40,26 +47,31 @@ public ProcessStateSystemInfo(String name) {
 
   // Visible for testing
   ProtobufSystemInfo.Section toProtobuf(MemoryMXBean memoryBean) {
-    ProtobufSystemInfo.Section.Builder builder = ProtobufSystemInfo.Section.newBuilder();
-    builder.setName(name);
+    ProtobufSystemInfo.Section.Builder protobuf = ProtobufSystemInfo.Section.newBuilder();
+    protobuf.setName(name);
+    addAttributeInMb(protobuf,"Max Memory (MB)", Runtime.getRuntime().maxMemory());
+    addAttributeInMb(protobuf, "Free Memory (MB)", Runtime.getRuntime().freeMemory());
     MemoryUsage heap = memoryBean.getHeapMemoryUsage();
-    addAttributeInMb(builder, "Heap Committed (MB)", heap.getCommitted());
-    addAttributeInMb(builder, "Heap Init (MB)", heap.getInit());
-    addAttributeInMb(builder, "Heap Max (MB)", heap.getMax());
-    addAttributeInMb(builder, "Heap Used (MB)", heap.getUsed());
+    addAttributeInMb(protobuf, "Heap Committed (MB)", heap.getCommitted());
+    addAttributeInMb(protobuf, "Heap Init (MB)", heap.getInit());
+    addAttributeInMb(protobuf, "Heap Max (MB)", heap.getMax());
+    addAttributeInMb(protobuf, "Heap Used (MB)", heap.getUsed());
     MemoryUsage nonHeap = memoryBean.getNonHeapMemoryUsage();
-    addAttributeInMb(builder, "Non Heap Committed (MB)", nonHeap.getCommitted());
-    addAttributeInMb(builder, "Non Heap Init (MB)", nonHeap.getInit());
-    addAttributeInMb(builder, "Non Heap Max (MB)", nonHeap.getMax());
-    addAttributeInMb(builder, "Non Heap Used (MB)", nonHeap.getUsed());
+    addAttributeInMb(protobuf, "Non Heap Committed (MB)", nonHeap.getCommitted());
+    addAttributeInMb(protobuf, "Non Heap Init (MB)", nonHeap.getInit());
+    addAttributeInMb(protobuf, "Non Heap Max (MB)", nonHeap.getMax());
+    addAttributeInMb(protobuf, "Non Heap Used (MB)", nonHeap.getUsed());
+    OperatingSystemMXBean os = ManagementFactory.getOperatingSystemMXBean();
+    setAttribute(protobuf,"System Load Average", format("%.1f%% (last minute)", os.getSystemLoadAverage() * 100.0));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define the locale to be used in this String operation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1449)